README.md

```diff
@@ -19,7 +19,7 @@
 
 ## What is CTI Types & Instances?
```

Comment:
Please also make sure that all occurrences of CTI Types & Instances are updated as well.